Frends.Community.HTMLToPdf/Frends.Community.HTMLToPdf.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+	  <TargetFrameworks>netstandard2.0;net6.0;net8.0</TargetFrameworks>
+	  <LangVersion>Latest</LangVersion>
+	  <Version>1.1.0</Version>
+	  <Authors>Frends</Authors>
+	  <Copyright>Frends</Copyright>
+	  <Company>Frends</Company>
+	  <Product>Frends</Product>
+	  <PackageTags>Frends</PackageTags>
+	  <PackageLicenseExpression>MIT</PackageLicenseExpression>
+	  <GenerateDocumentationFile>true</GenerateDocumentationFile>
+	  <Description>Frends task for converting HTML to PDF</Description>
+	  <PackageProjectUrl>https://github.com/CommunityHiQ/Frends.Community.HtmlToPdf</PackageProjectUrl>
+	  <RepositoryUrl>https://github.com/CommunityHiQ/Frends.Community.HtmlToPdf/tree/master/Frends.Community.HTMLToPdf</RepositoryUrl>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="FrendsTaskMetadata.json" Pack="true" PackagePath="/">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+
+  <ItemGroup>
+	<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+		<_Parameter1>$(MSBuildProjectName).Tests</_Parameter1>
+	</AssemblyAttribute>
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Haukcode.WkHtmlToPdfDotNet" Version="1.5.93" />
```

Comment:
This library uses LGPL 3.0 apparently, please update the task to comply with LGPL 3.0 requirements.